fitbenchmarking/fitbenchmarking
# 483 nist compute jacobians

#### Description of Work

Fixes #483 by computing analytic Jacobians (in a mathematica file), and putting them in a form that should be readable by the NIST parser (as `/NIST/???_difficulty/datafile/<problem_name>.jac`).  This will enable work to start on Issue #482 


#### Testing Instructions

1. Check each NIST problem has a Jacobian file
2. Check the format is easy to use
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
